mindmeld/augmentation.py

```diff
@@ -215,23 +232,27 @@ def __init__(self, batch_size, language, paths, path_suffix, resource_loader):
         PegasusForConditionalGeneration = _get_module_or_attr(
             "transformers", "PegasusForConditionalGeneration"
         )
-
-        model_name = "tuner007/pegasus_paraphrase"
-        torch_device = (
+        self.retain_entities = retain_entities
+        if self.retain_entities:
+            if not os.path.exists(PARAPHRASER_MODEL_PATH):
+                self._download_model()
+            model_name = PARAPHRASER_MODEL_PATH
+        else:
+            model_name = "tuner007/pegasus_paraphrase"
```

Comment:
Consider making this into a global variable? A comment or variable name explaining what this path is or where it comes from would be helpful